stable/prometheus-mongodb-exporter/templates/_helpers.tpl

```diff
@@ -51,4 +51,4 @@ Determine secret name, can either be the self-created of an existing one
 {{- else -}}
     {{ include "prometheus-mongodb-exporter.fullname" . }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
```

Comment:
Undo

Reply:
It's just a new line at the end of the file to not show the malformed file on GH.